src/templates/party-template.js

```diff
@@ -136,10 +137,45 @@ const PartyPage = ({ data: { partyYaml, allPeopleYaml } }) => {
               <h2 css={{ ...cssH2 }}>สมาชิกพรรค</h2>
               <p>จำนวนสมาชิกสภาผู้แทนราษฎร {partyYaml.total_member} คน</p>
 
-              <h2>ส.ส. บัญชีรายชื่อ / แบ่งเขต</h2>
-              <h2>เพศ</h2>
-              <h2>ช่วงอายุ</h2>
-              <h2>การศึกษา</h2>
+              <PartyProfileBar
+                dataset={[
+                  { label: "ส.ส. บัญชีรายชื่อ", value: 40, color: "white" },
+                  { label: "ส.ส. แบ่งเขต", value: 60, color: "black" },
```

Comment:
One small comment here.

On the user side, value needs to be pre-calculated to sum up to 100. It will be more convenient if it gets auto normalized inside the component.

Reply:
I'm not sure about this requirement so I didn't implement it 😅